### ISSUE TITLE
Having trouble with 3D reconstruction on own dataset

### ISSUE THREAD

**Issue by Uio96** [intel-isl/Open3D#2280]
**IMPORTANT: Please use the following template to report the bug.**
_____

**Describe the bug**
I tried to run the provided RGB-D reconstruction on a dataset. But every time, after I run the code for a while, there will be a termination, which said 
> joblib.externals.loky.process_executor.TerminatedWorkerError: A worker process managed by the executor was unexpectedly terminated. This could be caused by a segmentation fault while calling the function or by an excessive memory usage causing the Operating System to kill the worker. The exit codes of the workers are {SIGKILL(-9)}

I first assumed that there might be some problems with parallel. So I changed to a single worker. However, it still showed "killed" after some time. I also tried on another one. However, it did not work out. By the way, I once successfully ran the RGB-D reconstruction on other datasets with even more image pairs and the same resolution.  I basically followed the default parameter settings listed on the tutorial page but except the maximum depth value. 

**To Reproduce**

1. Download my dataset and put it into the dataset folder https://drive.google.com/file/d/1dKnUzIBih5WYnxsKDYDg2mZuLm4wtgcS/view?usp=sharing
2. python run_system.py dataset/Keyframe_Corbs_Desk_1_open3d/config.json --make --register --refine --integrate
3. See error

**Screenshots**
![error](https://user-images.githubusercontent.com/13082153/92134582-b219ac80-edd7-11ea-8a6d-b9004e43ebc4.png)

**Environment (please complete the following information):**

- Operating system: Ubuntu 16.04
- Python version: (Python 3.5)
- Open3D version: 0.9
- Is this remote workstation?: no
- How did you install Open3D?: conda

**Additional context**
I am not sure if it is necessary to change other parameter settings.

**Comment by Uio96** [comment]
I manually check the RGB-D info by visualizing it in a point cloud way. It seems fine to me. I am not sure if the reconstruction system has any other requirements for the depth info.

![demo](https://user-images.githubusercontent.com/13082153/92143637-8d2b3680-ede3-11ea-8451-562f50b5010a.png)

**Comment by s-agawane** [comment]
Have you been able to figure this out :/?

**Comment by VD2410** [comment]
Is this issue solved?

**Comment by theNded** [comment]
TLDR: Open3D does not work on CORBS.
Long version: 
CORBS is a tricky dataset: its depth images are not contiguous -- it has missing depth every other pixel. This means that our RGBD odometry would not work (properly) on this dataset; and with incorrect poses, TSDF integration will try to allocate as much memory as possible. In the end, a memory overflow will trigger a segfault.
